src/components/events/HardwareCaptionEncoderPanel.tsx

```
@@ -0,0 +1,121 @@
+      p_event_id: eventId,
+      p_encoder_type: encoderType,
+      p_rest_base_url: restBaseUrl.trim(),
+      p_rtmp_url: rtmpUrl.trim() || null,
+      p_channel_id: channelId.trim() || null,
+    });
+    setSaving(false);
+    if (error) {
+      toast.error(error.message || "Failed to save hardware encoder");
+      return;
+    }
+    toast.success("Hardware encoder identified. Deepgram captions will be burned into the RTMP feed.");
```

ESLint

[error] Replace `"Hardware·encoder·identified.·Deepgram·captions·will·be·burned·into·the·RTMP·feed."` with `⏎······"Hardware·encoder·identified.·Deepgram·captions·will·be·burned·into·the·RTMP·feed.",⏎····`

(prettier/prettier)